src/common/utils.js

```diff
@@ -49,29 +49,34 @@ Class.subClass = function( props )
 };
 
 // An enhanced DataView
-// Accepts an ArrayBuffer, typed array, or a length number
-function MemoryView( buffer, byteOffset = 0, byteLength )
+// Accepts an ArrayBuffer, another view (MemoryView / DataView / TypedArray), or a length number
+function MemoryView( buffer, byteOffset, byteLength )
```

Comment:
Added support for passing a MemoryView or DataView, which was a comment-only change (i.e., it already works).

(Used to avoid touching MemoryView.buffer in zvm.js.)